agent/engine/docker_task_engine_test.go

```diff
@@ -156,6 +156,7 @@ func TestBatchContainerHappyPath(t *testing.T) {
 	default:
 	}
 	eventsReported.Wait()
+	client.EXPECT().DescribeContainer(gomock.Any()).AnyTimes()
```

Comment:
According to the comment "Expect a bunch of steady state 'poll' describes when we trigger cleanup" there should still be calls to `DescribeContainer` during the remainder of the test.